schema.sh

```diff
@@ -1,7 +1,7 @@
-psql -d pacific -f database/schema-events.sql
-psql -d pacific -f database/schema-events-categories.sql
-psql -d pacific -f database/schema-users.sql
-psql -d pacific -f database/schema-user-preferences.sql
-psql -d pacific -f database/schema-users-event.sql
-psql -d pacific -f database/schema-friends.sql
-psql -d pacific -f database/schema-locations.sql
+psql -d pacific -U postgres -f database/schema-events.sql
```

Comment:
Going to make a PR to change this back since not everyone has a user named `postgres`